integrations/pom.xml

```diff
@@ -408,6 +400,31 @@
         <artifactId>jackson-module-jaxb-annotations</artifactId>
         <version>2.7.5</version>
       </dependency>
+      <dependency>
+        <groupId>org.apache.commons</groupId>
```

Comment:
Won't really represent the Jenkins behavior. Jenkins core version is used instead in runtime. In JTH this version will be picked instead

Reply:
So what the fix? or does it even matter :question:

Reply:
Pick your poison, one of dependencies declares a version of commons lang while it should not do so. I would rather prefer to add it to the Enforcer ignore list

Reply:
Shoot, I'll send a PR to plugin-pom 😅
or should we add it here?

Reply:
I see little difference in excluding vs dependency management.
At least with dependency management, we have control of which version is loaded.